app-nextjs-starter/pages/index.tsx

```diff
@@ -1,49 +1,50 @@
-import {GetServerSideProps, NextPage} from "next";
-import {isAppSessionQuery, Region, sessionCookieStore} from "@storyblok/app-extension-auth";
-import {authHandlerParams, endpointPrefix} from "@src/auth";
-import {Layout} from "@src/components/Layout";
-import {useEffect, useState} from "react";
-import {Stories} from "@src/components/Stories";
-import {isStories, Story} from "@src/Story";
+import { GetServerSideProps, NextPage } from 'next'
+import { authHandlerParams, endpointPrefix } from '@src/auth'
+import { Layout } from '@src/components/Layout'
+import { useEffect, useState } from 'react'
+import { StoryTable } from '@src/components/StoryTable'
+import { isStories, Story } from '@src/Story'
+import {
+  isAppSessionQuery,
+  sessionCookieStore,
+} from '@storyblok/app-extension-auth'
+import styles from '@src/styles/Index.module.css'
 
 type PageProps = {
-  userName: string
-  region: Region
-  spaceName: string
   spaceId: number
   userId: number
 }
 
-const fetchStories = (spaceId: number, userId: number): Promise<Story[]> => (
+const fetchStories = (spaceId: number, userId: number): Promise<Story[]> =>
   fetch(`api/stories?spaceId=${spaceId}&userId=${userId}`)
     .then((res) => res.json())
-    .then((res) => isStories(res) ? res : [])
-    .catch(error => {
+    .then((res) => (isStories(res) ? res : []))
+    .catch((error) => {
       console.error('Failed to fetch stories', error)
       return []
     })
-)
 
 const Home: NextPage<PageProps> = (props) => {
   const [stories, setStories] = useState<Story[]>([])
 
   useEffect(() => {
-    fetchStories(props.spaceId, props.userId)
-      .then(stories => setStories(stories))
+    fetchStories(props.spaceId, props.userId).then((stories) =>
+      setStories(stories),
+    )
   }, [props.spaceId, props.userId])
 
   return (
     <Layout>
-      <p>
-        Hello {props.userName} 👋
-      </p>
-      <p>
-        This is a <em>{props.region}</em> space.
-      </p>
-      <p>
-        These are the first {stories.length} stories on the {props.spaceName} space:
-      </p>
-      <Stories stories={stories}/>
+      <div className={styles.app}>
+        <span className={styles.app__text}>
+          Here is a list of the last {stories.length} updated stories on this
+          space:
+        </span>
+        <StoryTable
+          stories={stories}
+          className={styles['app__story-table']}
+        />
+      </div>
```

Comment:
Oh did we decide to remove `userName`, `spaceName`, and `region`? I think it's beneficial to have them here, so that users will know they can have those information at this point.
Even if we don't render them, we could still include them in `PageProps` and maybe `console.log(..)`. What do you think?

Reply:
As it was not being rendered anymore (after keeping the layout the same with the NuxtJS), I asked myself about keeping or not them... I preferred removing it since it was no use, however, using it for `logging` could be really nice.

 What are your thoughts on that @johannes-lindgren ?

Reply:
I think `userId`, `spaceId`, `region`, `accessToken` and `roles` are the most relevant properties. I haven't encountered a use case for `userName` or `spaceName` yet, so I'd be fine with omitting those. But if we want to print out `spaceName`, we could simply change the sentence to

```jsx
Here is a list of the last {stories.length} updated stories on the {spaceName} space:
```

The `region` is not displayed here, however, you can find them when we're instantiating the StoryblokJsClient. So we could skip logging in my opinion.